### ISSUE TITLE
Optimze ressource consumption of HTTP worker

### ISSUE THREAD

**Issue by berndruecker** [zeebe-io/zeebe-http-worker#30]
The current HTTP worker requires probably too much resources, especially compared to the "legacy" NodeJs one from the cloud. There is a lot of potential for optimization.

**Comment by saig0** [comment]
This is not surprising :sweat_smile: 

I guess the best results can be reached by a native image with GraalVM or Quarkus.
What do you think?

**Comment by berndruecker** [comment]
I agree - not surprising at all. Yeah, both are interesting directions to explore - maybe some others as well. For the moment it is not yet urgent, so I will not directly jump into it. But it would be indeed a good opportunity to play a bit with Quarkus :-)

**Comment by Klaus-Nji-sp** [comment]
Is resource consumption from Spring?  If so, have you guys also considered alternate frameworks such as Micronaut?

**Comment by berndruecker** [comment]
It is a couple of things. First you could definitely optimize the Spring Boot Setup. But we could also try some other frameworks. We deprioritized this for now and pretty sure will come back at a later stage.
